### PR TITLE
Some fixes for tests on Windows

### PR DESCRIPTION
**Pull request by Osyotr**

### CODE REVIEW

**Comment by Osyotr** [comment]
@biodranik would you kindly sync https://github.com/organicmaps/world_feed_integration_tests_data with upstream and cherry-pick remaining commits from https://github.com/mapsme/omim/commits/master? There is some discrepancy between test repo and test code.
Also https://github.com/mapsme/just_gtfs needs to be cloned into this organization (for https://github.com/mapsme/just_gtfs/pull/21).

**Review by biodranik — APPROVED** [review]
LGTM, @vng PTAL

File: generator/feature_sorter.cpp [hunk 1]
```diff
@@ -310,11 +311,24 @@ class FeaturesCollector2 : public FeaturesCollector
...
   public:
-    explicit TmpFile(std::string const & filePath) : FileWriter(filePath) {}
-    ~TmpFile() override { DeleteFileX(GetName()); }
+    explicit TmpFile(std::string const & filePath)
+      : m_writer(std::make_unique<FileWriter>(filePath)) {}
+
+    FileWriter & GetWriter()
+    {
+      return *m_writer;
+    }
```
Comment (biodranik) [inline generator/feature_sorter.cpp:324]:
```suggestion
    FileWriter & GetWriter() { return *m_writer; }
```

**Review by biodranik — APPROVED** [review]
LGTM, @vng can you please help with tests?

**Comment by Osyotr** [comment]
> Synced both, which commits should be picked from the master repo?

Everything from Mar 30, 2021 is missing in this repo but on a second look I don't see anything related to tests, so feel free to ignore me.

**Review by biodranik — APPROVED** [review]
Thanks!

**Comment by Osyotr** [comment]
@biodranik BTW are there any restrictions on using `std::filesystem`?

**Comment by biodranik** [comment]
@Osyotr yes:

> Filesystem library : Minimum deployment target:
macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0

https://developer.apple.com/xcode/cpp/

**Pull request merged**